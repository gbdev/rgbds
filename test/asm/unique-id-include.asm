REPT 2
	println "Within REPT"
	INCLUDE "unique-id-include.inc"
	println "Outside INCLUDE: \@"
ENDR

MACRO m1
	println "Within MACRO"
	INCLUDE "unique-id-include.inc"
	println "Outside INCLUDE: \@"
ENDM
	m1

println

REPT 2
	println "Within REPT: \@"
	INCLUDE "unique-id-include.inc"
	println "Outside INCLUDE: \@"
ENDR

MACRO m2
	println "Within MACRO: \@"
	INCLUDE "unique-id-include.inc"
	println "Outside INCLUDE: \@"
ENDM
	m2
