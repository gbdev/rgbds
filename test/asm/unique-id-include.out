Within REPT
Within INCLUDE: _u1
Outside INCLUDE: _u1
Within REPT
Within INCLUDE: _u2
Outside INCLUDE: _u2
Within MACRO
Within INCLUDE: _u3
Outside INCLUDE: _u3

Within REPT: _u4
Within INCLUDE: _u4
Outside INCLUDE: _u4
Within REPT: _u5
Within INCLUDE: _u5
Outside INCLUDE: _u5
Within MACRO: _u6
Within INCLUDE: _u6
Outside INCLUDE: _u6
